scripts/std-lib-regression.sh

```diff
@@ -2,14 +2,15 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
-# Deliberately not enabling this, since we expect a failure currently and are failing based on 'grep' later
-#set -eu
+set -eu
 
 # Test for platform
 PLATFORM=$(uname -sp)
-if [[ $PLATFORM == "Linux x86_64" ]]; then
+if [[ $PLATFORM == "Linux x86_64" ]]
```

Comment:
What auto-formatter are you using, and can we get it enabled in CI so we don't have whitespace wars?

Reply:
For bash scripts I just use `vim`. :)